src/alembic/versions/ce9a59922b43_tags_post_tags.py

```diff
@@ -0,0 +1,32 @@
+"""add blog_posts table (modified column name)
+
+Revision ID: ce9a59922b43
+Revises: e398258d0c63
+Create Date: 2024-08-09 04:25:35.100100
+
+"""
+from collections.abc import Sequence
+from typing import Union
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "ce9a59922b43"
+down_revision: str | None = "e398258d0c63"
+branch_labels: str | Sequence[str] | None = None
+depends_on: str | Sequence[str] | None = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("blog_posts", sa.Column("post_tags", sa.String(length=128), nullable=True))
+    op.drop_column("blog_posts", "tags")
+    # ### end Alembic commands ###
```

Comment:
let's combine these three migrations into just one for the tables (one migration per PR is the typical rule). Afterwards, double check that everything is the same as your `models.py`. Once that's the case, remove the commands "auto generated by Alembic"

Alembic will not do constraints correctly, so you'll have to make sure the ForeignKeyConstraint ids match manually. I'll double check afterwards though, so don't worry too much about it being perfect!